recipes/os-brick/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "0.3.2" %}
+
+package:
+  name: os-brick
+  version: {{ version }}
+
+source:
+  fn: os-brick-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/o/os-brick/os-brick-{{ version }}.tar.gz
+  md5: 5bc611df7e5af5194b5d6fc1ce92a743
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
Needs `pbr` in `build` [apparently]( https://travis-ci.org/conda-forge/staged-recipes/builds/161214081#L518 ).